packages/devtools_app/test/cpu_profile_model_test.dart

```diff
@@ -107,6 +123,7 @@ void main() {
           verboseName: 'all',
           category: 'Dart',
           rawUrl: '',
+          processedUrl: '',
```

Comment:
not your fault, but while you are modifying this fake data, could you also add the missing 'sourceLine' field?

Reply:
@kenzieschmoll where should I be adding the sourceLine field, I'm not sure I understand

Reply:
right below processedUrl. This should have been added when I added the sourceLine data but I missed it 🙃

Reply:
Going to handle this in https://github.com/flutter/devtools/issues/3974